tekton/pipeline_ref.go

```diff
@@ -0,0 +1,43 @@
+package tekton
+
+import tektonv1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1"
+
+type PipelineRef struct {
+	Name        string      `json:"name,omitempty"`
+	ResolverRef ResolverRef `json:"resolverRef"`
+}
+
+type ResolverRef struct {
+	Resolver string  `json:"resolver"`
+	Params   []Param `json:"params"`
+}
+
+type Param struct {
+	Name  string `json:"name"`
+	Value string `json:"value"`
+}
+
+func (pr PipelineRef) ToTektonPipelineRef() tektonv1.PipelineRef {
+	tektonParams := tektonv1.Params{}
+	for _, p := range pr.ResolverRef.Params {
```

Comment:
```suggestion
	for _, param := range pr.ResolverRef.Params {
```
Also, empty line before this one please.

Reply:
Done.